test/TestUtilities/GraphicsUtil.cpp

```diff
@@ -16,9 +16,8 @@ std::ostream& operator<<(std::ostream& os, const BlendMode& blendMode)
 
 std::ostream& operator<<(std::ostream& os, const Color& color)
 {
-    return os << "0x" << std::hex << color.toInteger() << std::dec << " (r=" << static_cast<int>(color.r)
-              << ", g=" << static_cast<int>(color.g) << ", b=" << static_cast<int>(color.b)
-              << ", a=" << static_cast<int>(color.a) << ")";
+    return os << "0x" << std::hex << color.toInteger() << std::dec << " (r=" << +color.r << ", g=" << +color.g
+              << ", b=" << +color.b << ", a=" << +color.a << ")";
```

Comment:
I don't think this is an improvement. The unary `operator+` is hard to spot, and the rules here are much less obvious than a `static_cast`. It seems that we are optimizing for conciseness compared to readability, and I'm not confident that is a worthwhile goal/metric.

Reply:
Yeah, I'd consider this a hack, as it works around a technical "issue" and destroys the readability of the code. A large percentage of C++ developers won't be able to tell you *what* `+color.r` does and even fewer *why" it's not just `color.r`.

Reply:
I agree with vittorio and eXpl0it3r here. As much as I hate verbose `static_cast`, I'd probably use something like `toUint8` or so, which is at least somewhat intuitive.

One thing where I did find unary `+` useful though is for `enum class` and converting to their underlying type. So you could have similar conciseness as in C++98, but wouldn't need type-unsafe enums.

Reply:
@Bromeon: Note that C++23 provides `std::to_underlying` (https://en.cppreference.com/w/cpp/utility/to_underlying) which clearly expresses that intent. I'd rather have a SFML polyfill of that compared to using `operator+`.

Reply:
Fair. This is perhaps too clever.